experiments/pure_mcts/create_experiments_from_config.py

```diff
@@ -0,0 +1,168 @@
+        dir_name = [item.lower() for item in dir_name]
+
+    if hyphenated or shorten:
+        old_dir_name = dir_name
+        dir_name = []
+        for item in old_dir_name:
+            if item == False or item == "False" or item == "false":
+                dir_name.append("f")
+            elif item == True or item == "True" or item == "true":
+                dir_name.append("t")
+            else:
+                dir_name.append(item)
+
+    dir_name.append(str(num_games))
+    if hyphenated:
+        dir_name = "-".join(dir_name)
+    else:
+        dir_name = "_".join(dir_name)
+
+    if shorten:
+        if len(dir_name) > 32:
+            dir_name = dir_name[:24] + "-" + secrets.token_hex(4)[:7]
+
+    return dir_name
+
+
+def run_games(num_games: int, save_plots: bool = True) -> tuple[list[str], list[str]]:
+    # Get the directory containing the yaml file
+    current_file = os.path.abspath(__file__)
+    current_dir = os.path.dirname(current_file)
+
+    with open(f"{current_dir}/config.yaml", "r") as f:
+        hyperparameters = yaml.load(f, Loader=yaml.FullLoader)
+
+    for game_name in hyperparameters["game_name"]:
+        for num_rollouts in hyperparameters["num_rollouts"]:
+            for num_rollouts_2 in hyperparameters["num_rollouts_2"]:
+                # Set all the hyperparameters, fixing the values of the ones
+                # we've already set
+                game_command = []
+                for key, value in hyperparameters.items():
+                    if key == "game_name":
+                        game_command += ["--" + key, game_name]
+                    elif key == "num_rollouts":
+                        game_command += ["--" + key, str(num_rollouts)]
+                    elif key == "num_rollouts_2":
+                        game_command += ["--" + key, str(num_rollouts_2)]
+                    else:
+                        game_command += ["--" + key, str(value)]
+
+                all_results = []
+                all_errors = []
+
+                dir_name = get_directory_name_from_command(game_command, num_games)
+                directory_path = f"{experiments_directory}/{dir_name}"
+
+                container = "ghcr.io/alignmentresearch/polygames:1.4.1-runner"
+
+                single_command = (
+                    f"python /polygames/experiments/pure_mcts/run_given_experiment.py "
+                    f"{shlex.join(game_command)} --SPECIAL_num_games {num_games} "
+                    f"--SPECIAL_save_plots {save_plots} --SPECIAL_directory_path {directory_path}"
+                )
+
+                # For testing
+                # subprocess.run(shlex.split(single_command))
+
+                experiment_name = get_directory_name_from_command(
+                    game_command, num_games, hyphenated=True, shorten=True
+                )
+
+                print("experiment name:", experiment_name, "length:", len(experiment_name))
+
+                docker_command = (
```

Comment:
```suggestion
                # The docker command runs `run.sh`, which we create from the `commands` variable below.
                docker_command = (
```

Add comment explaining